engine/pokemon/experience.asm

```diff
@@ -31,6 +31,8 @@ CalcLevel:
 	ret
 
 CalcExpAtLevel:
+; BUG: Experience underflow for level 1 Pokemon with Medium-Slow growth rate (see docs/bugs_and_glitches.md)
```

Comment:
IMO the function documentation is more important than a note about some bug with that function. So I'd format this sort of bug as such (which also avoids the blank line):
```
CalcExpAtLevel:
; (a/b)*n**3 + c*n**2 + d*n - e
; BUG: Experience underflow for level 1 Pokemon with Medium-Slow growth rate (see docs/bugs_and_glitches.md)
	ld a, [wBaseGrowthRate]
	add a
```